### PR TITLE
Docs: edit Configure standard options

### PR DESCRIPTION
**Pull request by imatwawana**
Content additions and copy edits
To be merged into branch from PR https://github.com/grafana/grafana/pull/84225

TO DO (after approval)

- [x] Delete images from local and upload to admin; update links in md file
- [x] Merge into PR https://github.com/grafana/grafana/pull/84225

### CODE REVIEW

**Comment by imatwawana** [comment]
@jdbaldry - I've just added you as a reviewer to look at that build error. I don't know what to do there.

**Comment by baldm0mma** [comment]
@imatwawana also, just a heads, the dashboard people put up a fix for that unit-clearing issue you brought up. Fix is [here](https://github.com/grafana/grafana/pull/84449).

**Comment by jdbaldry** [comment]
The error looks unrelated to your changes and I believe you can ignore it.

**Comment by jdbaldry** [comment]
Please ensure that you when you merge, you "Squash and merge" (should be the default, the alternative is "Rebase and merge") so that the commits containing the images are squashed out of the final commit made to the `main` branch.

**Comment by imatwawana** [comment]
> Please ensure that you when you merge, you "Squash and merge" (should be the default, the alternative is "Rebase and merge") so that the commits containing the images are squashed out of the final commit made to the `main` branch.

I always do another set of commits to remove the images as well, so I think that should take care of that issue, no?

**Comment by jdbaldry** [comment]
> > Please ensure that you when you merge, you "Squash and merge" (should be the default, the alternative is "Rebase and merge") so that the commits containing the images are squashed out of the final commit made to the `main` branch.
> 
> I always do another set of commits to remove the images as well, so I think that should take care of that issue, no? Also, this is getting merged into another branch (not `main`).

Unless the commit that adds the images is rewritten away (either by squashing or dropping the commit), then the binary file will always be in the Git index which bloats it. Otherwise, Git wouldn't be able to let you go back to the commit where it was added (because the data wouldn't be there).

File: docs/sources/panels-visualizations/configure-standard-options/index.md [hunk 1]
```diff
@@ -25,84 +25,94 @@ weight: 60
 
 # Configure standard options
 
-The data model used in Grafana, namely the [data frame](https://grafana.com/developers/plugin-tools/introduction/data-frames), is a columnar-oriented table structure that unifies both time series and table query results. Each column within this structure is called a _field_. A field can represent a single time series or table column.
+**Standard options** in the panel editor pane let you change how field data is displayed in your visualizations. Options that you apply don't change the data, they just change how Grafana _displays_ the data.
```
Comment (codeincarnate) [inline docs/sources/panels-visualizations/configure-standard-options/index.md:28]: Moving away from a technical description of data frames here seems like a very good thing 😄 🥳

File: docs/sources/panels-visualizations/configure-standard-options/index.md [hunk 2]
```diff
@@ -25,84 +25,94 @@ weight: 60
...
+You can also paste a native emoji in the **Unit** drop-down and select it as a custom unit:
+
+![A thumbs up emoji as a custom unit](/media/docs/grafana/panels-visualizations/custom_unit_thumbsup_v11.0.png)
+
+![A time series visualization using custom thumbs up emoji units](/media/docs/grafana/panels-visualizations/thumbsup_panel_v11.0.png)
 
 #### String units
 
-Grafana can sometimes be too aggressive in parsing strings and displaying them as numbers. To configure Grafana to show the original string value, create a field override and add a unit property with the `String` unit.
+Sometimes Grafana is too aggressive in parsing strings and displaying them as numbers. To configure Grafana to show the original string value, select **Misc > String** in the **Unit** drop-down.
```
Comment (codeincarnate) [inline docs/sources/panels-visualizations/configure-standard-options/index.md:89]: Maybe _interpreting_ would make more sense than _parsing_? Seems a bit less technical, but certainly not my area of expertise here 😄

**Review by codeincarnate — APPROVED** [review]
Gave it a read-through and looks great!

File: docs/sources/panels-visualizations/configure-standard-options/index.md (hunk 1 above)
Comment (imatwawana) [reply docs/sources/panels-visualizations/configure-standard-options/index.md:28]: Big shoutout to @baldm0mma for his help here!

File: docs/sources/panels-visualizations/configure-standard-options/index.md (hunk 2 above)
Comment (imatwawana) [reply docs/sources/panels-visualizations/configure-standard-options/index.md:89]: I like it. Will do!

Comment (imatwawana) [inline docs/sources/panels-visualizations/configure-standard-options/index.md:89]:
```suggestion
Sometimes Grafana is too aggressive in interpreting strings and displaying them as numbers. To configure Grafana to show the original string value, select **Misc > String** in the **Unit** drop-down.
```

**Review by baldm0mma — APPROVED** [review]
Great!

**Pull request merged**